### ISSUE TITLE
Couldn't pass parameters to testng.xml from command line with Maven

### ISSUE THREAD

**Issue by HariHaraBhuvanesh** [cbeust/testng#2339]
### TestNG Version
> 7.1.0

I followed the instructions specified by @krmahadevan in [stackoverflow](https://stackoverflow.com/questions/50266606/how-to-pass-parameter-to-testng-xml-from-command-line-with-maven) and in his [blog](https://rationaleemotions.wordpress.com/2017/09/29/dynamic-parameterization-in-testng/) to pass the parameters from command line to testng.xml with maven but it did not work either. I couldn't comment as I'm bit new to stackoverflow and dont have sufficient reputation to do so. Hence creating this ticket to fix the issue.

### Test case sample
```java
@Test
@Parameters("name")
public void hello(String name) {
	System.err.println("Hello " + name + " !");
}
```

**testng.xml**

```xml
<?xml version="1.0"?>
<!DOCTYPE suite SYSTEM "http://testng.org/testng-1.0.dtd">
<suite name="sample_suite" verbose="1" parallel="false" thread-count="2">
    <test name="sample_test">
        <parameter name="name" value="Krishnan"/>
        <classes>
            <class name="ParameterisedSampleTestClass"/>
        </classes>
    </test>
</suite>
```

**mvn command**
>mvn clean test -Dname=Hari
![image](https://user-images.githubusercontent.com/49117146/86527138-a41fdc00-beb9-11ea-862d-9611ad27364f.png)

**output**
>Hello Krishnan !
![image](https://user-images.githubusercontent.com/49117146/86527144-bf8ae700-beb9-11ea-9182-a46b4bbe037e.png)

### Expected behavior
As per @krmahadevan comment [here](https://stackoverflow.com/questions/50266606/how-to-pass-parameter-to-testng-xml-from-command-line-with-maven) and in his [blog](https://rationaleemotions.wordpress.com/2017/09/29/dynamic-parameterization-in-testng/) parameter value passed from command line should be passed to the test. In the example provided command line value "Hari" should be passed as value instead of "Krishnan" that is hardcoded in testng.xml

So in short expected output is 
>"Hello Hari!" and not "Hello Krishnan !"

### Actual behavior
Instead of command line value "Hari", value "Krishnan" is passed to test and it print as 
>"Hello Krishnan !" instead of "Hello Hari !"

### Is the issue reproductible on runner?

- [ ] Shell
- [x] Maven
- [ ] Gradle
- [ ] Ant
- [x] Eclipse
- [x] IntelliJ
- [x] NetBeans

**Comment by krmahadevan** [comment]
This is already fixed as part of https://github.com/cbeust/testng/issues/2238 and will be available as part of `7.2.0` release which is yet to be released.
